tests/rpc-handler.test.ts

```diff
@@ -1,20 +1,27 @@
-import { RPCHandler } from "../dist/cjs/src/rpc-handler";
-import { HandlerConstructorConfig } from "../src/handler";
-import { networkRpcs } from "../dist/cjs/src/constants";
 import { JsonRpcProvider } from "@ethersproject/providers";
+import { HandlerConstructorConfig } from "../dist/tests/mocks/handler";
+import { RPCHandler } from "../dist/tests/mocks/rpc-handler";
+import { networkRpcs } from "../dist/tests/mocks/constants";
+
+// mocks have replaced axios with fetch api for testing
+// as axios was having troubles with adapter recognition
 
 export const testConfig: HandlerConstructorConfig = {
   networkId: 100,
   autoStorage: false,
   cacheRefreshCycles: 3,
+  rpcTimeout: 1500,
```

Comment:
Isn't the timeout set to 500? I presume this is a configurable value.

Reply:
yeah added it as a configurable for those who are playing around on unpopular networks